Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:6.5
+
+WORKDIR /app
+COPY package.json /app/
+RUN npm install --loglevel warn
+RUN npm config set unsafe-perm true
+COPY . /app/
+
+CMD ["npm", "test"]
```

Comment:
This was a great idea. Is there a way for you to document how to run the tests in docker?